src/fixtures/ShopwareDataFixtures.ts

```diff
@@ -0,0 +1,36 @@
+import { test as base } from '@playwright/test';
+import type { FixtureTypes } from '../types/FixtureTypes';
+
+export interface ShippingMethod {
+    id: string;
+}
+
+export interface ShopwareDataFixtureTypes {
+    ShippingMethod: ShippingMethod;
+}
+
+export const test = base.extend<NonNullable<unknown>, FixtureTypes>({
+
+    ShippingMethod: [
```

Comment:
For `PaymentMethod` we could do the same

Reply:
Yes, I intend to change all of https://github.com/shopware/acceptance-test-suite/blob/ba108ea999e25e825c46ec308202b247d077c089/src/fixtures/DefaultSalesChannel.ts#L52-L59
to an fixture. Then you can overwrite them as needed.